### DISCUSSION TITLE
is your samsung galaxy fold 3 powerful enough?

### DISCUSSION THREAD

**Discussion by elvisisvan** [phoenixbyrd/Termux_XFCE#23] [General]
is it powerful enough to handle all your tasks smoothly?

i think using a mobile device as complete replacement for an actual laptop is a bit of a stretch, do you really don't need a laptop? how long would it take for you to set up each time you start working?

**Comment by jagalindo** [comment]
I'm pretty sure that if you only plan to code and or do office work. It should be enough.  I've been using it on a s20fe 5g with no issues. Now on s23u, the experienced is mostly the same.

**Reply by jagalindo** [reply]
The single problem for me is the hid, I mean, having a good keyboard mouse and the alike to be productive. Also, do not rely on wireless dex

**Comment by phoenixbyrd** [comment]
I've been doing this since the note 9, depends on your needs if this is going to work for you or not. If you're stuck in an ecosystem you can't escape from and replicate here then it won't replace anything for you.
